CrossSolar/Controllers/AnalyticsController.cs

```diff
@@ -51,9 +51,39 @@ public async Task<IActionResult> Get([FromRoute]string panelId)
 
         // GET panel/XXXX1111YYYY2222/analytics/day
```

Comment:
when app is running GET to endpoint panel/AAAA1111BBBB2222/analytics/day is not working